app/views/homepage/popular_links/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render "govuk_publishing_components/components/heading", {
```

Comment:
I think we should use a [Form fieldset](https://components.publishing.service.gov.uk/component-guide/fieldset) component to group each of the link sections together, which would be helpful for assistive technology, rather than having a form full of individual inputs. Each of the headings in that case would be a `<legend>` element.